### PR TITLE
Issue  #29442 resolved

### PR DESCRIPTION
**Pull request by ps-19**
https://github.com/kubernetes/website/issues/29442
Link is updated.

### CODE REVIEW

File: content/en/docs/contribute/generate-ref-docs/_index.md [hunk 1]
```diff
@@ -9,4 +9,4 @@ reference guides.
 
 To build the reference documentation, see the following guide:
 
-* [Generating Reference Documentation Quickstart](/docs/contribute/generate-ref-docs/quickstart/)
+* [Generating Reference Documentation Quickstart](https://github.com/kubernetes/website/blob/f7c51d8db526199df2aba67ee5ea7a0e01c8e996/content/en/docs/contribute/generate-ref-docs/quickstart.md)
```
Comment (tengqm) [inline content/en/docs/contribute/generate-ref-docs/_index.md:12]: ？

File: content/zh/docs/contribute/generate-ref-docs/_index.md [hunk 2]
```diff
@@ -22,5 +22,5 @@ To build the reference documentation, see the following guide:
 本节的主题是描述如何生成 Kubernetes 参考指南。
 要生成参考文档，请参考下面的指南：
 
-* [生成参考文档快速入门](/zh/docs/contribute/generate-ref-docs/quickstart/)
+* [生成参考文档快速入门](https://github.com/kubernetes/website/blob/f7c51d8db526199df2aba67ee5ea7a0e01c8e996/content/en/docs/contribute/generate-ref-docs/quickstart.md)
```
Comment (tengqm) [inline content/zh/docs/contribute/generate-ref-docs/_index.md:25]: ？

**Comment by reylejano** [comment]
Hi @ps-19, I'm closing this PR for now since the link in this PR is incorrect and the PR needs to be split on a per-language / localization basis
Feel free to re-open this PR when the link is updated correctly and the PR is only for one language / localization

/close

**Pull request closed without merging**